src/main/java/uk/gov/hmcts/darts/cache/token/service/Token.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.hmcts.darts.cache.token;
+package uk.gov.hmcts.darts.cache.token.service;
 
 import jakarta.servlet.http.HttpServletRequest;
 import lombok.EqualsAndHashCode;
```

Comment:
Can you check this annotation please - do you want to include `validate` in hashCode and equals?
Only asking because there is `@EqualsAndHashCode.Include` on token and sessionId fields, then @EqualsAndHashCode on the class itself, so looks like it needs to be tidied up a bit

Reply:
Originally was intending to use the object as a key in redis but I ended up keying on getId string. Removed the annotations for now so we use refrence equality